scripts/configure.js

```diff
@@ -323,6 +323,10 @@ const getConfig = (() => {
             "metro.config.js": {
               source: path.join(testAppPath, "example", "metro.config.js"),
             },
+            ".gitignore": fs
+              .readFileSync(path.join(testAppPath, "example", ".gitignore"),{
+                encoding: "utf-8",
+              }),
```

Comment:
Since we're not making any changes to this file, we can just copy it:
```suggestion
            ".gitignore": {
               source: path.join(testAppPath, "example", ".gitignore"),
             },
```
Same below.